src/equiv.js

```diff
@@ -126,6 +126,48 @@ QUnit.equiv = (function() {
 					parentsB.pop();
 					return true;
 				},
+				
+				"set": function( b, a ) {
+					var aArray, bArray;
+					
+					// b could be an object literal here
```

Comment:
Inspecting the code further, I think the check is necessary, since `objectType()` is only run on `a`, and never on `b` prior to executing the function `callbacks['set']` / `callbacks['map']`.